moto/resourcegroups/responses.py

```diff
@@ -22,8 +22,13 @@ def create_group(self):
         description = self._get_param("Description")
         resource_query = self._get_param("ResourceQuery")
         tags = self._get_param("Tags")
+        configuration = self._get_param("Configuration")
         group = self.resourcegroups_backend.create_group(
-            name=name, description=description, resource_query=resource_query, tags=tags
+            name=name,
+            description=description,
+            resource_query=resource_query,
+            tags=tags,
+            configuration=configuration,
```

Comment:
This should have it's own test, to confirm that we can create a group that already contains a Configuration

Reply:
done